Covid Track/Covid Track/Utilities/Extension/Views/Cell+ResubaleView.swift

```diff
@@ -10,11 +10,11 @@ import Foundation
 import UIKit
 
 protocol ReusableView {
-  static var defaultResuseIdentifier: String { get }
+  static var defaultReuseIdentifier: String { get }
 }
 
 extension ReusableView where Self: UIView {
-  static var defaultResuseIdentifier: String {
+  static var defaultReuseIdentifier: String {
     return NSStringFromClass(self)
```

Comment:
Please follow the proper naming convention.

Reply:
Apple uses `open func register(_ cellClass: AnyClass?, forCellReuseIdentifier identifier: String)` function to register tableview cell. I have updated name variable name to - reuseIdentifier